recipe/meta.yaml

```diff
@@ -1,33 +1,41 @@
 {% set name = "certipy" %}
-{% set version = "0.1.3" %}
+{% set version = "0.2.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 695704b7716b033375c9a1324d0d30f27110a28895c40151a90ec07ff1032859
+  sha256: fef1f3d8819ee29c4c67719171c988302823dfe0b6cfbb47d249f374809ba05e
 
 build:
-  noarch: python
   number: 0
-  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  script: "{{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv"
+  script_env:
+    - SETUPTOOLS_SCM_PRETEND_VERSION={{version}}
   entry_points:
     - certipy = certipy.command_line:main
 
 requirements:
   host:
-    - python >=3.3
+    - python
     - pip
+    - wheel
+    - setuptools >=64
+    - setuptools-scm >=7
   run:
-    - python >=3.3
+    - python
     - pyopenssl
```

Comment:
```suggestion
```
Looks like it dropped its dependency on pyopenssl
https://inspector.pypi.io/project/certipy/0.2.2/packages/8d/8c/04b8edd906f298d4c23c0db8394784073aedced8559cd55d2de8480774af/certipy-0.2.2.tar.gz/certipy-0.2.2/pyproject.toml#line.41